face.go

```diff
@@ -28,15 +29,16 @@ type Recognizer struct {
 // Face holds coordinates and descriptor of the human face.
 type Face struct {
 	Rectangle  image.Rectangle
+	Shapes     []image.Point
 	Descriptor Descriptor
 }
 
 // Descriptor holds 128-dimensional feature vector.
 type Descriptor [128]float32
 
 // New creates new face with the provided parameters.
-func New(r image.Rectangle, d Descriptor) Face {
-	return Face{r, d}
+func New(r image.Rectangle, s []image.Point, d Descriptor) Face {
```

Comment:
Would be better to make another function `NewWithShape`, to avoid breaking API

Reply:
Fixed